src/app/[slug]/page.tsx

```diff
@@ -1,26 +1,33 @@
 import { getAllPaths, getFileForRoute, sluggify } from "@/content/getData";
+import { getRawAvailableOptions } from "@/lib/api";
 import ConceptsBoxWrapper, {
   ConceptsBox,
 } from "@/modules/Concepts/ConceptsBox";
 import ConceptsExampleWrapper, {
   ConceptsExample,
 } from "@/modules/Concepts/ConceptsExample";
 import FooterNav from "@/modules/Concepts/FooterNav";
-import PageAssessment from "@/modules/DynamicAssessment/PageAssessment";
+import PageOptions from "@/modules/Options/PageOptions";
+import { OptionArg, OptionSide, OptionType } from "@/types/option";
 import { MDXRemote } from "next-mdx-remote/rsc";
 
 export default async function Page({ params }: { params: { slug: string } }) {
   const data = await getFileForRoute(params.slug);
-
-  const option = {
-    side: params.slug === "buying" ? "long" : "short",
-    type: "call",
-  };
+  const { data: rawOptionsData } = await getRawAvailableOptions();
 
   if (!data) {
     return <p>Page not found</p>;
   }
 
+  const displayOptions = [
+    "buying-call",
+    "buying-put",
+    "selling-call",
+    "selling-put",
+  ].includes(params.slug);
```

Comment:
Might be easier to directly use the PageAssessment component in the mdx files, like that you wouldn't have to check the URL to determine what type of option to show